registry/spec.md

```diff
@@ -1491,71 +1632,1350 @@ An attempt to delete all versions MUST generate an error.
 A `DELETE /GROUPs/ID/RESOURCEs/ID/versions` without a body MUST delete all
 versions (except the latest) of the Resource.
 
+## CloudEvents Registry
+
+The CloudEvents Registry is a universal catalog and discovery metadata format
+and metadata service API for messaging and eventing schemas and metaschemas and
```

Comment:
would love to know of anyone else had trouble parsing this due to all of the "and"s :-)